features/support/env.rb

```diff
@@ -17,4 +17,16 @@
 require 'cucumber/rails'
 Capybara.default_selector = :css
 ActionController::Base.allow_rescue = false
+
+require 'database_cleaner/active_record'
 DatabaseCleaner.strategy = :truncation
+
+Around do |scenario, block|
+  DatabaseCleaner.cleaning(&block)
+end
+
+# 'Fixnum' and 'Bignum' are deprecated since ruby-3.2 (both replaced by 'Integer')
+# But 'machinist' still uses 'Fixnum', so we have to use workaround:
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.2')
+  class Fixnum < Integer; end
```

Comment:
I thought that was much before 3.2... but anyway, its a good idea.

Reply:
oh, maybe only do it if `defined?(Machinist)` ... not everybody uses it. I believe its an optional dependency

Reply:
This file is only included when we're running cucumber tests 😅

And since in our cucumber tests we're always testing integration with Machinist, I think we need this this fix too.

Or can we run cucumber tests without testing Machinist integration? 🤔